baklava/src/utils/keybinds/worker.ts

```diff
@@ -2,9 +2,13 @@ import { isMainThread, parentPort } from 'worker_threads';
 import globkey from 'globkey';
 
 if (!isMainThread) {
+    let prev_keys = ['', '']
```

Comment:
wait this line wasn't here? how did typescript not throw a bazillion errors!?

Reply:
before, rust handled the prev_keys logic and all that stuff, so this update hands that over to TS.

Reply:
ahh makes sense...